src/DurableTask.Netherite.AzureFunctions/NetheriteProviderStartup.cs

```diff
@@ -1,26 +1,28 @@
-﻿// Copyright (c) Microsoft Corporation.
+// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Collections.Concurrent;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
 // Reference: https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection
-[assembly: Microsoft.Azure.Functions.Extensions.DependencyInjection.FunctionsStartup(
-    typeof(DurableTask.Netherite.AzureFunctions.NetheriteProviderStartup))]
+[assembly: WebJobsStartup(typeof(DurableTask.Netherite.AzureFunctions.NetheriteProviderStartup))]
 
 namespace DurableTask.Netherite.AzureFunctions
 {
-    using System.Collections.Concurrent;
-    using Microsoft.Azure.Functions.Extensions.DependencyInjection;
-    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
-    using Microsoft.Extensions.DependencyInjection;
-
-    class NetheriteProviderStartup : FunctionsStartup
+    class NetheriteProviderStartup : IWebJobsStartup
     {
-        public override void Configure(IFunctionsHostBuilder builder)
+        public void Configure(IWebJobsBuilder builder)
         {
 #if !NETCOREAPP2_2
             builder.Services.AddSingleton<IDurabilityProviderFactory, NetheriteProviderFactory>();
 #else
             builder.Services.AddSingleton<IDurabilityProviderFactory, NetheriteProviderPseudoFactory>();
 #endif
+            builder.AddDurableTask();
```

Comment:
I'm not a blocking reviewer here, but I'm surprised about this change. Why is this here now?

Reply:
I definitely have very limited knowledge about this code. I am just trying to follow pattern used here  https://github.com/Azure/azure-functions-durable-extension/blob/dev/src/WebJobs.Extensions.DurableTask/DurableTaskWebJobsStartup.cs. Please share all the corrections needed here and I will update the PR.
cc [sebastianburckhardt](https://github.com/sebastianburckhardt)

Reply:
I think the `builder.AddDurableTask()` is superfluous here... I would expect this to already be taken care of by the existing startup code in the durable task extension.

Reply:
Done.